### PR TITLE
Support discoverd URI schemes including tcp://

### LINKED ISSUE #7: DISCOVERD env var format
When running discoverd in a docker container, docker exposes where it is running in an environment variable formatted like `tcp://172.17.0.36:1111`. Whereas go-discoverd expects the format to be `172.17.0.36:1111`. 

Is it worth tweaking this behaviour to be able to handle either style for compatibility with docker?

### PR DESCRIPTION
**Pull request by john-griffin**
Fixes #7

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -338,6 +339,8 @@ func NewClient() (*Client, error) {
 	addr := os.Getenv("DISCOVERD")
 	if addr == "" {
 		addr = "127.0.0.1:1111"
+	} else {
+		addr = strings.Replace(addr, "tcp://", "", 1)
```
Comment (titanous) [inline client.go]: [`TrimPrefix`](http://golang.org/pkg/strings/#TrimPrefix) would be even better.

**Comment by titanous** [comment]
Please [accept the Developer's Certificate of Origin](https://github.com/flynn/flynn-guide/blob/master/CONTRIBUTING.md#developers-certificate-of-origin) in your commit.

**Comment by john-griffin** [comment]
Good call! Updated.

**Comment by titanous** [comment]
Awesome, thanks!

**Pull request merged**